integration_tests/CMakeLists.txt

```
@@ -858,6 +858,7 @@ RUN(NAME format_34 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
 RUN(NAME format_35 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
 RUN(NAME format_36 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
 RUN(NAME format_37 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
+RUN(NAME format_38 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc fortran)
```

```suggestion
RUN(NAME format_38 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
```

You can remove the `fortran` flag for now, to fix the failing CI and can implement the feature is in the `asr_to_fortran.cpp` in a subsequenct PR. Or if you wish you can do it in this PR also.